src/SPECS/hootenanny.spec.in

```diff
@@ -18,7 +18,8 @@ BuildRequires:  git glpk-devel graphviz libicu-devel
 BuildRequires:  hootenanny-words = 1.0.0
 BuildRequires:  libspatialite-devel
 BuildRequires:  log4cxx-devel nodejs-devel opencv-devel
-BuildRequires:  postgresql-devel >= 9.1
+#BuildRequires:  postgresql-devel >= 9.1
+BuildRequires:  postgresql93-devel
```

Comment:
Do we need to handle the case where we are doing an upgrade to an existing install running postgresql 9.2?